ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -44,6 +44,13 @@ spec:
           --pyxis-url $(params.pyxis_url) \
           --cert-project-id $(params.cert_project_id)
 
+        CERTIFICATION_STATUS=$(cat cert_project | jq '.certification_status')
```

Comment:
This task is used in different pipelines, and not for all of them status "In Progress" is true.
We will update certification status in different pipelines, assigning different values to it.
Could you create a generic task containing `update-cert-project-data` script? This task should accept the certification status as parameter, so that it could be easily reused :)

Reply:
Updated!